frontend/src/components/MapView/Layers/WMSLayer/index.tsx

```diff
@@ -26,6 +27,8 @@ const WMSLayers = ({
     <Source
       id={`source-${id}`}
       type="raster"
+      // refresh tiles every time date changes
+      key={moment(queryDate).format(DEFAULT_DATE_FORMAT)}
```

Comment:
```suggestion
      key={new Date(queryDate).toISOString().slice{0, 10)}
```
Let's try to avoid `moment` as per #1063.